pkg/git/editor_test.go

```diff
@@ -87,20 +86,6 @@ func TestGetDefaultGitEditor(t *testing.T) {
 		defaultIDE, _ := getDefaultGitEditor()
 		assert.Equal(t, defaultIDE, "value")
 	})
-
-	t.Run("no GIT_EDITOR or EDITOR defaults to vi", func(t *testing.T) {
```

Comment:
What do you think of changing this test to `"no GIT_EDITOR falls back to EDITOR or OS fallback"` and change L101 to 
```
fallbackEditor := os.Getenv("EDITOR")
if fallbackEditor == "" {
    assert.Equal(t, "vi", newEditor)
} else {
    assert.Equal(t, fallbackEditor, newEditor)
}
```

Reply:
yeah sure, that works for me too!